src/scope/core/pipelines/registry.py

```diff
@@ -86,5 +89,21 @@ def _register_pipelines():
         PipelineRegistry.register(config_class.pipeline_id, pipeline_class)
 
 
+def _initialize_registry():
+    """Initialize registry with built-in pipelines and plugins."""
+    # Register built-in pipelines first
+    _register_pipelines()
```

Comment:
This will fail on the non-gpu devices, because it will try to import the pipelines which have dependencies on CUDA. We could either try/catch for each import, or make the CUDA import optional in each pipelines.But I thought that maybe even better is to treat this plugin stuff as an opportunity to use only those default pipelines which fit your hardware requirements. In particular:
1. Display Krea only if you run > 5090 (don't display it on 4090)
2. Display only Passthrough if you run on a non-gpu machine (in future, we'll also display Decart API)

I've drafted a PR and would be interested in your thoughts: https://github.com/daydreamlive/scope/pull/243